src/reducer.js

```diff
@@ -0,0 +1,14 @@
+import { LP_API_ACTION } from './actions'
+import { get } from './utils'
+
+export default function (state={}, { type, payload }) {
+  if (type !== LP_API_ACTION) return state
+  const { key, status } = payload
+  return { ...state, [key]: status }
+}
+
+export function selectStatus (key, state, slice='api') {
```

Comment:
Good for now, ideally we'd want `state` to be the last argument here since its the data on which we are operating. Our options for doing that are:
1. being super smart and handling `selectStatus(key, state)` & `selectStatus(key, slice, state)`
2. allow the implementor to configure the slice when creating the middleware.

I would prob opt for 2, but this doesn't have to be done now.